sa-export.ps1

```diff
@@ -1,32 +1,35 @@
-
 $type = "csv" # type can be database and csv
```

Comment:
Maybe we can move this to a resource script instead? That prevents using both provisioning and service automation.